pages/manual-content/sitemap.njk

```diff
@@ -0,0 +1,16 @@
+---
+permalink: /sitemap.xml
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  {% for page in collections.all %}
+    {% if not sitemap.urls[page.url | url ].exclude %}
+      <url>
+        <loc>https://digital.ca.gov{{ page.url | url }}</loc>
```

Comment:
Can the url string come from setting or global variables file? We've been thinking about code portability in the context of ODI publishing.